crates/bitwarden-core/Cargo.toml

```diff
@@ -45,6 +45,7 @@ rand = ">=0.8.5, <0.9"
 reqwest = { workspace = true }
 schemars = { workspace = true }
 serde = { workspace = true }
+serde_bytes = ">=0.11.17, <0.12"
```

Comment:
Do we need to move this to the workspace? Both crypto and core use it at this point.

Reply:
Yeah I think that sounds like a good idea here.